app/models/post.rb

```diff
@@ -124,6 +124,6 @@ def member
   end
 
   def rendered_description
-    RDiscount.new(description, :autolink)
+    RDiscount.new(description || '', :autolink)
```

Comment:
Should this method already return safe rendered html, rather than returning an initialized RDiscount instance? Are we using anything else than rendering html from this instance?